### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by dimqua** [nutritionfactsorg/daily-dozen-android#59]
Please consider to include the app into F-Droid. F-Droid is an installable catalogue of free and open source software for the Android platform. It would be beneficial for both the app's popularity and for those users who prefer don't use Google Play store.

Currently, however, the app don't complies with their [inclusion policy](https://f-droid.org/wiki/page/Inclusion_Policy):

> We cannot build apps using proprietary tracking/analytic dependencies like Crashlytics and Firebase. Please talk to upstream about an untainted build flavor (either using a FLOSS analytics software like ACRA or by removing non-free deps completely).

Are you interested in this? Thanks.

**Comment by pouladpld** [comment]
I am also a fdroid user and look forward seeing this app there. Considering the charitable nature of this FLOSS software, fdroid is the right place for it.

**Comment by FHulot** [comment]
Adding my vote for this getting into F-Droid!

I know nothing about writing these apps, so forgive me if this is naive, but

`grep -r [Cc]rashlytics  *`

on the repo gets only four hits:

- app/build.gradle:    compile('com.crashlytics.sdk.android:crashlytics:2.6.8@aar') {
- app/proguard-rules.pro:# Fabric/Crashlytics - from https://docs.fabric.io/android/crashlytics/dex-and-proguard.html
- app/src/main/java/org/nutritionfacts/dailydozen/DailyDozenApplication.java:import com.crashlytics.android.Crashlytics;
- app/src/main/java/org/nutritionfacts/dailydozen/DailyDozenApplication.java:            Fabric.with(this, new Crashlytics());

Here is a bit more of that last file:
`        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        } else {
            Fabric.with(this, new Crashlytics());
        }
`
I am wishfully thinking that  Crashlytics might  actually be used only once, and only for debugging. So...easily removed/replaced??

**Comment by paranoicogm** [comment]
Adding my vote as well!

**Comment by c64bob** [comment]
Yes, please publish a version of this on f-droid! How can we help?

**Comment by alibitek** [comment]
A solution for this issue is in progress in pull request #81

What has been done?
Written the necessary metadata file, built the app, signed the build and tested it on a device.
  
What needs to be done?
We need the support of the project maintainer.
Then I can start the merging process in the F-Droid's GitLab data repository.

**Comment by FHulot** [comment]
Has the maintainer responded?

**Comment by alibitek** [comment]
@FHulot Unfortunately, no 😞 
I'd really like to get the #81 PR merged, but without the support from the project maintainer I cannot.
The only alternative option is to submit my own fork to the F-Droid repository.

**Comment by slavick** [comment]
@alibitek I will review your pull request this upcoming weekend.

**Comment by slavick** [comment]
Closing this issue as PR https://github.com/nutritionfactsorg/daily-dozen-android/pull/81 has been merged.

**Comment by alibitek** [comment]
Daily Dozen has been merged into fdroiddata GitLab repository: https://gitlab.com/fdroid/fdroiddata/merge_requests/2746
Awaiting for the the app to be built and included in the official repository enabled in the F-Droid app. 
When this happens the https://f-droid.org/app/org.nutritionfacts.dailydozen/ URL will work.
